### DISCUSSION TITLE
3/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#102] [Daily Sprint]
Qué hice ayer
Qué hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
Ayer participe de la clase
Hoy nos reuniremos para ver cómo se sigue.
Impedimento, tiempos

**Comment by rpgrca** [comment]
Ayer se me pasó la clase, pensé que era a las 20. En cuanto al TP terminé creo que todos los modelos de la app, también armé una base para el repositorio y la barra de navegación inferior con las 4 cosas que tendríamos que tener, el catálogo está crudo todavía. Hoy voy a estar en la weekly, también voy a continuar un poco más con lo de la app para seguir avanzando.
